x-pack/plugins/security_solution/public/timelines/components/timeline/body/column_headers/header/helpers.ts

```diff
@@ -11,15 +11,18 @@ import { Sort, SortDirection } from '../../sort';
 
 interface GetNewSortDirectionOnClickParams {
   clickedHeader: ColumnHeaderOptions;
-  currentSort: Sort;
+  currentSort: Sort[];
 }
 
 /** Given a `header`, returns the `SortDirection` applicable to it */
 export const getNewSortDirectionOnClick = ({
```

Comment:
I think we don't need the `!` after `onColumnSorted` in x-pack/plugins/security_solution/public/timelines/components/timeline/body/column_headers/header/index.tsx Line 37